.ci/create_vm.gitlab-ci.yml

```diff
@@ -0,0 +1,161 @@
+#
+# Create VM with RHEL9
+#
+create:rhel9:vm:
+  stage: create vm
+  tags:
+    - docker-throttled
+  script:
+    - echo "TEST HAS STARTED"
+    - apt-get update -y
+    - apt-get install curl -y
+    - echo "Creating virtual machine with RHEL9"
+    - 'curl --location "http://qapi.datto.net/v1/agent" --header "Content-Type: application/json" --data "{\"os\":\"rhel9\",\"agent_type\":\"\",\"region\":\"nwk\",\"ttl\":1}" > createVmOutput.txt'
+    - cat createVmOutput.txt
+    - OUTPUT=$(cat createVmOutput.txt)
+    - echo $OUTPUT
```

Comment:
printing 2 times creatVmOutput is not needed

Reply:
done